scripts/firedrake-install

```diff
@@ -1585,35 +1587,33 @@ if mode == "install":
         f90 = os.path.join(compilerbin, "mpif90")
         compiler_env = create_compiler_env(cc, cxx, f90)
 
+    pip = Pip(python, "-m", "pip", parallel_env=compiler_env, blas_env=blas)  # !!!
```

Comment:
I think that it's a little odd to have multiple `Pip` objects hanging around. Couldn't we rely on static/class methods?